### ISSUE TITLE
Build error in Github Actions

### ISSUE THREAD

**Issue by blacklightpy** [obsidian-html/obsidian-html#610]
I got this error while trying to build in Github Actions using ubuntu-latest. 

```
running: 
	 rsync -av /home/runner/work/vaultofknowledge/vaultofknowledge/vaultofknowledge/ /tmp/tmpca0pw8iv --exclude /.obsidian --exclude /.trash --exclude /.DS_Store --exclude /.git
Traceback (most recent call last):
  File "/home/runner/.local/bin/obsidianhtml", line 8, in <module>
    sys.exit(main())
  File "/home/runner/.local/lib/python3.10/site-packages/obsidianhtml/__init__.py", line 30, in main
    ConvertVault()
  File "/home/runner/.local/lib/python3.10/site-packages/obsidianhtml/controller/ConvertVault.py", line 42, in ConvertVault
    tmpdir = Actor.Optional.copy_vault_to_tempdir(pb)  # isort: skip DO NOT REMOVE reference "tmpdir ="  or the folder will be immediately removed!
  File "/home/runner/.local/lib/python3.10/site-packages/obsidianhtml/core/Actor.py", line 19, in copy_vault_to_tempdir
    tmpdir = CreateTemporaryCopy(source_folder_path=pb.paths["obsidian_folder"], pb=pb)
  File "/home/runner/.local/lib/python3.10/site-packages/obsidianhtml/core/CopyVault.py", line 39, in CreateTemporaryCopy
    copy_tree_rsync(source_folder_path.as_posix(), tmpdir.name, exclude=pb.gc("exclude_glob"), verbose=pb.gc("copy_vault_to_tempdir_follow_copy"))
  File "/home/runner/.local/lib/python3.10/site-packages/obsidianhtml/core/CopyVault.py", line [92](https://github.com/blacklightpy/vaultofknowledge/actions/runs/4393310900/jobs/7693631234#step:3:93), in copy_tree_rsync
    print("Copy succeeded %d %s %s" % (p.returncode, output.decode("ascii").replace("\\n", "\n"), error))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 20628: ordinal not in range(128)
Error: Process completed with exit code 1.
```
On a windows image it goes upto the second-pass, but still fails.

```
< FEATURE: PROCESS ALL: Done
	> SECOND PASS HTML
		Root used for glob pattern expansion: html

		Excluded Folders (configured):
- .git
- __src
- md
- obs.html


		Excluded Files (configured):
- .gitignore
- favicon.ico
- not_created.html


		Excluded Folders (expanded from glob patterns and found):
[]


		Excluded Files (expanded from glob patterns and found):
[]

Error: Process completed with exit code 1.
```

I was using the latest master build from github, using pip.

**Comment by dwrolvink** [comment]
I am having the same issue all of a sudden, seems to be fixed by #611 for me

**Comment by blacklightpy** [comment]
It is now fixed 👍
